www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -284,14 +284,21 @@ _**Note:** The directories above are generally located in the Android SDK ROOT._
 
 #### macOS and Linux
 
-On a Mac or Linux, with a text editor, create or modify the `~/.bash_profile` file.
+On a Mac older then Catalina or on Linux, create or modify the `~/.bash_profile` file for e.g. with a text editor.
+On macOS Catalina and newer, create or modify the `~/.zprofile`, because it's using ZSH instead of Bash.
 
 To set an environment variable, add a line that uses `export` like so (substitute the path with your local installation):
 
 ```bash
 export ANDROID_HOME=/Development/android-sdk/
 ```
 
+on macOS
+
+```zsh
+export ANDROID_HOME=~/Library/Android/sdk/
+```
+
```

Comment:
This section may be unnecessary, as the previous line instructs to "substitute the path with your local installation."

The example path, `/Development/android-sdk/`, as far as I am aware is not a standard or default path, even on Linux. It was only for an example. I believe it was intended to account for users who set up their Android development environment with `cmdline-tools` rather than Android Studio.

Reply:
Maybe something like this is clearer that it's an example placeholder?

```bash
export ANDROID_HOME=/path/to/android-sdk/
```

Reply:
Could there be not suggestions for the different OSs? It is more helpful, if the Guide already takes you to the right place of your SDK.
I overhead the part "substitute the path with your local installation". But the example `ANDROID_HOME=/path/to/android-sdk/`would make it more clear

Reply:
If we wanted to suggest for different OS, we could update the entire section like this?

```
    Add the following line to your shell's profile to set up the `ANDROID_HOME` environment variable.
    
    **macOS:**
    
    ```bash
    export ANDROID_HOME=~/Library/Android/sdk
    ```
    
    **Linux:**
    
    ```bash
    export ANDROID_HOME=~/Android/Sdk
    ```
    
    The above examples point to the standard paths where the Android SDK is typically stored. Depending on your environment configuration, the path may differ. Be sure to confirm that the path exist or adjust them to match your local installation.
    
    After setting the `ANDROID_HOME` environment variable, update the `PATH` variable to include directories containing various Android-related binaries.
    
    ```bash
    export PATH=$PATH:$ANDROID_HOME/platform-tools/
    export PATH=$PATH:$ANDROID_HOME/cmdline-tools/latest/bin/
    export PATH=$PATH:$ANDROID_HOME/build-tools/
    export PATH=$PATH:$ANDROID_HOME/emulator/
    ```
```

The above:

* Provide example `export` lines for macOS and Linux, pointing to standard paths.
* Mention that these are **standard paths**, but that the actual path **may differ** depending on your environment configuration.
* Remind you to **confirm that each path exists** and to **adjust paths as needed** to match your installation.

I also revised the instructions above the `export PATH` section to improve flow and make it easier to follow. While it’s helpful to provide cut-and-paste examples, it’s equally important that users review each step, as environments can vary.

If you use the above, make sure to fix the indentation from what I posted it. I had to indent the content to prevent GH from rendering the markdown.

Reply:
Ah sorry, I didn't saw Windows has it's own section :)